src/Params.ml

```diff
@@ -13,11 +13,16 @@ module Format = Format_
 open Migrate_ast
 open Fmt
 
-type exp_wrap = Fmt.t -> Fmt.t
```

Comment:
Why remove that type ?
It was used to show that `get_exp_wrap` was returning a wrapping function and wasn't one itself.
But it was not implemented that way so it may makes sense.

Reply:
Yes, and also it was used only once, if we want to abstract that and use a new type, `Fmt.ml` would be a better place to define it.